xdcs-server/src/main/java/pl/edu/agh/xdcs/db/dao/LogLineDao.java

```diff
@@ -29,4 +30,15 @@
                 .setParameter("t", Optional.ofNullable(to).orElse(DbUtils.MAX_INSTANT))
                 .getResultList();
     }
+
+    @SuppressWarnings("unchecked")
+    public List<LogLineEntity> findByAgents(String taskId, AgentEntity agent) {
+        return entityManager.createQuery("select ll from LogLine ll " +
+                "where ll.agent in :agent and " +
```

Comment:
in?

Reply:
miała być tablica agentów przekazana, poprawie